### PR TITLE
Removed overlapping Fixes Issue#2492

### PR DESCRIPTION
**Pull request by Pranshu1902**
Removed overlap and made the buttons better

image

### CODE REVIEW

**Comment by Pranshu1902** [comment]
Fixes #2492

**Comment by nihal467** [comment]
oekwpfew
@gigincg @Pranshu1902 
test failed, unable to see the button

**Comment by Pranshu1902** [comment]
@nihal467 I meant the pagination buttons, they were overlapping before
Screenshot (333)

**Comment by rithviknishad** [comment]
@Pranshu1902 deploy preview is failing. merge the latest develop branch.

**Comment by nihal467** [comment]
@Pranshu1902 @gigincg the pagination is missing, can you share me the patient URL, which you used

**Comment by Pranshu1902** [comment]
http://localhost:4000/facility/7a560e62-6aae-4293-8f1a-0ba840e3449f/patient/192fdee9-397f-49bd-833a-10f700416da8/consultation/d794ba54-da42-4238-8ba9-0b628ca60069

**Comment by nihal467** [comment]
test approved

File: src/Components/Facility/Consultations/DailyRoundsList.tsx [hunk 1]
```diff
@@ -65,14 +65,14 @@ export const DailyRoundsList = (props: any) => {
...
-  } else if (dailyRoundsListData.length > 0) {
+  } else if (dailyRoundsListData.length) {
     roundsList = dailyRoundsListData.map((itemData, idx) => {
       const telemedicine_doctor_update =
         itemData.created_by_telemedicine ||
         itemData.last_updated_by_telemedicine;
 
       return (
-        <div key={`daily_round_${idx}`} className="w-full">
+        <div key={`daily_round_${idx}`} className="w-ful">
```
Comment (khavinshankar) [inline src/Components/Facility/Consultations/DailyRoundsList.tsx]: is `w-ful` a valid tailwind class 😕? please do check this one

File: src/Components/Common/Pagination.tsx [hunk 2]
```diff
@@ -127,13 +127,13 @@ const Pagination = (props: PaginationProps) => {
...
         <div
-          onClick={(e) => handleChangePage(e, "prev")}
+          onClick={(e) => handleChangePage("prev")}
           className="relative inline-flex items-center px-4 py-2 border border-gray-300 text-sm leading-5 font-medium rounded-md text-gray-700 bg-white hover:text-gray-500 focus:outline-none focus:ring-primary focus:border-primary-300 active:bg-gray-100 active:text-gray-700 transition ease-in-out duration-150"
         >
           Previous
         </div>
         <div
-          onClick={(e) => handleChangePage(e, "next")}
+          onClick={(e) => handleChangePage("next")}
```
Comment (khavinshankar) [inline src/Components/Common/Pagination.tsx]:
```suggestion
          onClick={() => handleChangePage("next")}
```

File: src/Components/Common/Pagination.tsx [hunk 3]
```diff
@@ -127,13 +127,13 @@ const Pagination = (props: PaginationProps) => {
     <div className="mx-auto mb-4">
       <div className="flex-1 flex justify-between sm:hidden">
         <div
-          onClick={(e) => handleChangePage(e, "prev")}
+          onClick={(e) => handleChangePage("prev")}
```
Comment (khavinshankar) [inline src/Components/Common/Pagination.tsx]:
```suggestion
          onClick={() => handleChangePage("prev")}
```

File: src/Components/Common/Pagination.tsx [hunk 4]
```diff
@@ -102,7 +102,7 @@ const Pagination = (props: PaginationProps) => {
     return (
       <button
         disabled={disabled}
-        onClick={(e) => handleChangePage(e, label.toLowerCase())}
+        onClick={(e) => handleChangePage(label.toLowerCase())}
```
Comment (khavinshankar) [inline src/Components/Common/Pagination.tsx]:
```suggestion
        onClick={() => handleChangePage(label.toLowerCase())}
```

**Review by khavinshankar — CHANGES_REQUESTED** [review]
please make the requested changes

**Comment by Pranshu1902** [comment]
@khavinshankar done

**Review by khavinshankar — APPROVED** [review]

**Review by gigincg — APPROVED** [review]

**Pull request merged**